cnf-certification-test/identifiers/identifiers.go

```diff
@@ -161,6 +162,17 @@ The label value is not important, only its presence.`,
 		bestPracticeDocV1dot4URL+" Section 3.5.4",
 		TagExtended)
 
+	TestAffinityRequired = AddCatalogEntry(
+		"affinity-required",
+		common.LifecycleTestKey,
+		`Checks that affinity rules are in place if AffinityRequired: true labels are set on Pods, Deployments, and StatefulSets.`,
```

Comment:
Can we remark here what I said about "true" is string but not a boolean?

Reply:
Done.